android/src/main/kotlin/com/gojuno/commander/android/Adb.kt

```diff
@@ -53,12 +54,12 @@ fun connectedAdbDevices(): Observable<Set<AdbDevice>> = process(listOf(adb, "dev
 
 fun AdbDevice.log(message: String) = com.gojuno.commander.os.log("[$id] $message")
 
-fun AdbDevice.installApk(pathToApk: String): Observable<Unit> {
+fun AdbDevice.installApk(pathToApk: String, timeout: Pair<Int, TimeUnit> = 5 to MINUTES): Observable<Unit> {
```

Comment:
Let's reduce default to 2 minutes, sounds reasonable?

Reply:
OK.
Both composer and commander are before version 1.0.0 so I assume that it is acceptable that users which currently need timeout between 2 and 5 minutes will have to adjust their configurations.

Reply:
yep, that's the rationale, once we get to 1.x will strictly follow semantic versioning!